common-components/index.md

```diff
@@ -116,21 +115,51 @@ Read the [Cloud Infrastructure and Platform Services wiki](https://educationgovu
 
-[description]
+##### Enterprise API Management (EAPIM) Platform
+The EAPIM platform provides a central place to host and discover APIs. It is a secure and fully accredited service that makes it easy to both publish and consume APIs.
 
-__Documentation__
+EAPIM has been developed using Microsoft Azure’s API gateway, and the solution is deployed within the [CIP hosting platform](#cip) with an expert team supporting the infrastructure.
 
-__Support__
+Below is an overview of some of the key benefits.
 
-__Roadmap__
+**Making it easier to share data**
+- Developers can securely publish their APIs quickly in one place using the developer portal
+- Data consumers can view all the APIs available and request access to data
+- The service scales quickly in response to changing demand
 
-__Further guidance__
+**Reducing the admin for developer teams**
+- Developers can host their APIs quickly by using automated features to build, test and publish. Features include automated on-boarding, templates, tested global policy controls, and identity and access management
+- Once an APIs is hosted, there’s no need for developers to perform any platform maintenance
+- Security accreditation is already in place
+
+**Cost savings**
+- EAPIM has a flexible subscription model, which means you won’t pay for more hosting space than you use
+- By moving APIs to EAPIM, we are reducing the number of duplicate API platforms across the estate
+
+##### Integration middleware
+In addition to the API platform, integration middleware services are also available. These services use APIs to improve data sharing between multiple systems.
+
+##### Benefits of using middleware to connect systems:
+The middleware technology improves access to siloed data and can deliver it between multiple systems in near real-time. By integrating systems in this way, we get better visibility of data, reduce the need to manually input data into multiple systems, increase efficiency, and reduce the risk of data error.
+
+##### Key documents
+Read the [Operating Model](https://educationgovuk.sharepoint.com/sites/lveesfa00073/API%20Management/Forms/AllItems.aspx?id=%2Fsites%2Flveesfa00073%2FAPI%20Management%2FPlatform%20Documentation%2FService%20Operating%20Model%20%28SOM%29%2FCurrent%20Version&viewid=00000000%2D0000%2D0000%2D0000%2D000000000000) for more information about the solution design, security, availability, support and governance. If you need access to this document, please contact Matt Morgan.  
+
+##### Support
+The service is supported internally by a central team of specialists within the Cloud Infrastructure and Platform Services Team. To report an issue with the service, contact the service desk; the team are available from 08:00 – 22:00 hrs Monday to Friday, excluding English public holidays.  
 
+##### Roadmap
+Over the next 18 months, these services are evolving in two key areas:
+- Driving adoption
+    - Onboarding more APIs onto the platform
```

Comment:
Any examples? Need to build trust, e.g. services x, y, z have already on-boarded and service a, b, c are in the pipeline.